docs/source/demo_callback.json

```diff
@@ -0,0 +1,58 @@
+{
+  "files": [
+    {
+      "primaryFilePath": "LNII/sbest/sbest-2023-0526-Rename/Spatial_example-2023-0526-Rename1/LNII/F13-TS005.czi",
+      "fileMetadata": {
+        "TBD": "TBD"
+      },
+      "imageRecords": [
```

Comment:
maybe s/imageRecords/imageSets/g ?

Reply:
A bit pedantic, but isn't "imageSet" already plural? Unless we have multiple sets of images, I think it should be imageSet.